bibliopixel/animation/remote/server.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, q_send, q_recv):
         self.q_send = q_send
         self.q_recv = q_recv
 
+    @classmethod
+    def shutdown_server(cls):
+        log.info('Shutting down UI server.')
+        func = flask.request.environ.get('werkzeug.server.shutdown')
+        if func is None:
```

Comment:
This should just be a warning - or even it could just return False if it didn't manage to shut down.